.rubocop.yml

```diff
@@ -121,6 +121,8 @@ Style/PredicateName:
 Style/GuardClause:
   Exclude:
     - 'lib/active_fedora/autosave_association.rb'
+    - 'lib/active_fedora/file_configurator.rb'
```

Comment:
Can you say why you added the three new line-item exclusions here rather than whipping the code into shape? Is it because there are too many violations to handle at this time?